compiler/rustc_codegen_llvm/src/lib.rs

```diff
@@ -232,6 +232,29 @@ impl CodegenBackend for LlvmCodegenBackend {
                 }
                 println!();
             }
+            PrintRequest::StackProtectorStrategies => {
+                println!(
+                    r#"Available stack protector strategies:
```

Comment:
4th option, `none` missing it's description

Reply:
Good catch, added now.